### DISCUSSION TITLE
Split query text in subparts for highlightning the result?

### DISCUSSION THREAD

**Discussion by nowrap** [digitalfortress-tech/typeahead-standalone#68] [Ideas]
Hello,
right now the highlightning marks only the complete query term in the results.

Maybe it is an idea to split the query term, if it has blanks and other characters. Could be both be configurable.

Example (random generated data):
![grafik](https://github.com/digitalfortress-tech/typeahead-standalone/assets/1099533/8d7e2e8a-5219-4f03-8fbc-5a4dcef61de1)

It would be nice to mark also the hits in the other two results.

Regards
nowrap

**Comment by niketpathak** [comment]
Hello, good idea. Thank you for the suggestion. A future version will include an option to be able to customise highlighting in multiple ways, i.e. highlight only terms that appear together, or highlight each individual term or both.

**Reply by suasgn** [reply]
Great, it is shared the same idea #65

Will this be released sooner?
